tools/cleanupTarchives.pl

```diff
@@ -221,11 +291,20 @@ sub selectTarchives {
 
 
 =pod
-Function that will determine the tarchive basename from the ArchiveLocation stored in the database. 
-It will, among other things, get rid of the _digit part that was inserted in the past by the tarchiveLoader.
-Input: - $tarchive      = ArchiveLocation that was stored in the tarchive table of the database.
-Output:- $tarBasename   = tarchive basename to use to look for duplicate tarchive in the tarchive library directory of the filesystem
+
+=head3 getTarchiveBasename($tarchive)
+
+Function that will determine the DICOM archive basename from the C<ArchiveLocation>
+stored in the database. It will, among other things, get rid of the C<_digit part>
+that was inserted in the past by the C<tarchiveLoader>.
+
+INPUT: C<ArchiveLocation> that was stored in the C<tarchive> table of the database.
+
+RETURNS: the DICOM archive basename to use to look for duplicate DICOM archives in
```

Comment:
...basename to use __when looking__ for duplicate DICOM...

Reply:
done